### PR TITLE
Add stress test for camera-to-texture upload

### PR DESCRIPTION
**Pull request by kainino0x**
This will be used as a perf test for Chrome.

### CODE REVIEW

File: sdk/tests/extra/texture-from-camera-stress.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+const urlOptions = wtu.getUrlOptions();
+const uploadsPerFrame = urlOptions.uploadsPerFrame ? urlOptions.uploadsPerFrame : 400;
+debug('');
+debug(`Testing with ${uploadsPerFrame} uploads per frame`);
+
+function startTestLoop() {
+    requestAnimationFrame(startTestLoop);
+    for (let i = 0; i < uploadsPerFrame; ++i) {
+        gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGB, gl.RGB, gl.UNSIGNED_BYTE, video);
```
Comment (jdashg) [inline sdk/tests/extra/texture-from-camera-stress.html]: It would be nice to get numbers for RGBA and texSubImage as well.

**Comment by jdashg** [comment]
Oops, I see you explicitly tagged kenrussell for review. Revert if desired!

**Comment by kainino0x** [comment]
@jdashg No problem at all! Thanks for the review!

**Comment by kenrussell** [comment]
Looks good to me too. Sorry for my slow review. No problem @jdashg for merging this earlier. :)

**Pull request merged**